style.css

```diff
@@ -63,3 +61,194 @@ li {
+  margin: 0 auto;
+  text-align: center;
+}
+
+h2 {
+  font-size: 40px;
+  color: #172b4d;
+}
+
+.recent-works h2:first-child::after {
+  content: '';
+  display: block;
+  position: relative;
+  height: 4px;
+  width: 48px;
+  margin: 0 auto;
+  top: 20px;
+  z-index: -1;
+  border-radius: 24px;
+  background: #36b37e;
+}
+
+.myworks-card {
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  padding: 0;
+  margin: 1em auto;
+  width: 327px;
+  height: 474px;
+  border: 1px lightgray solid;
+  border-radius: 5px;
+}
+
+.myworks-thumbnail {
+  grid-column: 1/-1;
+  grid-row: 2/3;
+  overflow: hidden;
+  box-sizing: border-box;
+}
+
+.myworks-thumbnail img {
+  width: 100%;
+  min-height: 220px;
+  object-fit: cover;
+  background-color: #d0d9d4;
+  border-radius: 2px;
+}
+
+.work-title {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 1.5em;
+  color: #3a4a42;
+  text-align: center;
+  grid-column: 1/3;
+  grid-row: 3/4;
+}
+
+.work-languages li {
+  background: #ebf0ee;
+  padding: 0.6em;
+  margin: 0 0.5em;
+  border-radius: 4px;
+  font-size: 12px;
+  font-weight: 600;
+}
+
+.attribute-list li {
+  background-color: #fff;
+  color: #36b37e;
+  font-size: 14px;
+  font-weight: 600;
+  padding: 1em 0.5em;
+  margin: 1em 0.5em;
+  border-radius: 5px;
+}
+
+.myworks-card > .work-languages > li {
+  background: #ebf0ee;
+  font-family: Inter, sans-serif;
+  font-size: 16px;
+  font-weight: 600;
+  padding: 0.5em;
+  margin: 0 0.5em;
+  border-radius: 4px;
+}
+
+.work-languages {
+  display: flex;
+  justify-content: space-around;
+  list-style: none;
+  padding: 0.5em;
+}
+
+button {
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
+  border: none;
+  border-radius: 4px;
+  width: fit-content;
+  align-self: center;
+  background: #36b37f;
+  color: white;
+  font-weight: 500;
+  font-size: 17px;
+  letter-spacing: 2px;
+  padding: 12px;
+}
+
+button:hover {
+  background: #36b37f;
+  box-shadow: 1px 8px 16px rgba(54, 179, 127, 0.24);
+  border-radius: 4px;
+}
+
+button:active {
+  background: #008552;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  margin-top: 2em;
+  background: url(./images/containerbg1.svg) no-repeat 100% 0%, url(./images/containerbg2.svg) no-repeat 0 300px;
+  padding: 2em;
+}
+
+.about-me p {
+  font-size: 20px;
+  color: #42526e;
+  font-weight: 400;
+  line-height: 32px;
+}
+
+.container-aboutme {
+  width: 90%;
+  max-width: 900px;
+  margin: 0 auto;
+}
+
+.my-attributes-ul {
+  list-style: none;
+  display: flex;
+  width: max-content;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+}
+
+.attribute {
+  display: flex;
+  flex-direction: column;
+  border: 1px lightgray solid;
+  padding: 40px;
+  border-radius: 5px;
+  margin-bottom: 2em;
+  background: #ebf0ee;
+  width: 327px;
+  height: 352px;
+}
```

Comment:
- [ ] There are a couple of things to note when working on projects and microverse project requirements tend to provide some insights to this which i think would be best for you to start taking note of. in this css class selector styling you provided a fixed width in `px`, doing this has a tendency to disrupt the overall design when the screen size gets reduced and as such the below image is what you have as a webpage, as you can see the fixed width of `327px` your added in this `.attribute` styling now affects your headline section causing there to be extra space on a `375px` screen please kindly fix this issue by making use of `%` which are responsive to screen size and not fixed.
![header-s-6](https://user-images.githubusercontent.com/89055502/235274237-7479d8aa-711a-4f83-8448-6f72fef3fea3.png)

- The below image is now what your header looks like when the fixed width is removed
![header-s-6-2](https://user-images.githubusercontent.com/89055502/235274403-0fa2d8fb-1b64-463a-ac34-36c2c3aac5bf.png)
- [ ] Kindly take note of this and please fix it before your next review request :+1: 
- [ ] Also you would now notice from the above image that your hamburger icon has not been styled correctly all along. i will leave a review on that particular icon i just wanted to note it to you now